src/irmin-pack/unix/file_manager.ml

```diff
@@ -797,4 +797,61 @@ struct
     let chunk_start_idx = pl.chunk_start_idx in
     let chunk_num = pl.chunk_num in
     cleanup ~root ~generation ~chunk_start_idx ~chunk_num
+
+  let create_one_commit_store config ~generation ~new_store_root
+      ~latest_gc_target_offset ~suffix_start_offset commit_key =
+    let open Result_syntax in
+    let root = Irmin_pack.Conf.root config in
+    (* Step 1. Copy the dict *)
+    let src_dict = Irmin_pack.Layout.V4.dict ~root in
+    let dst_dict = Irmin_pack.Layout.V4.dict ~root:new_store_root in
+    let* () = Io.copy_file ~src:src_dict ~dst:dst_dict in
+    (* Step 2. Create an empty suffix and close it. *)
+    let* suffix =
+      Suffix.create_rw ~root:new_store_root ~overwrite:false
+        ~auto_flush_threshold:1_000_000 ~auto_flush_procedure:`Internal
```

Comment:
It doesn't impact the functioning of the code, but this isn't using the `auto_flush_threshold` from the `config`.

Reply:
Yeah, it really does not matter, as we write nothing in the suffix file. I would have exposed a `touch` function in io, if there was one provided by the unix/sys libraries.